### ISSUE TITLE
Addition of pod labels without modifying selectors

### ISSUE THREAD

**Issue by rob8714** [kubernetes-sigs/kustomize#4546]
Currently the `labels` field does not add pod labels if `includeSelectors` is set to `false`.  It seems that the options available to add pod labels are by either setting `includeSelectors` to `true` or using `commonLabels`, but I would like to be able to do this without updating selectors. It's also possible to add the labels with a patch, but it would be nice if the `labels` field could handle it

For example:

```yaml
# kustomization.yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

labels:
  - pairs:
      someName: someValue
      owner: alice
      app: bingo
    includeSelectors: false


resources:
- deploy.yaml
- service.yaml


# deploy.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: example

# service.yaml
apiVersion: v1
kind: Service
metadata:
  name: example
```

Requested build output:

```yaml
apiVersion: v1
kind: Service
metadata:
  labels:
    app: bingo
    owner: alice
    someName: someValue
  name: example
---
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: bingo
    owner: alice
    someName: someValue
spec:
  template:
    metadata:
      labels:
        app: bingo
        owner: alice
        someName: someValue
```

**Comment by KnVerey** [comment]
/triage accepted

@natasha41575 what are your thoughts on what the syntax for this could be?

**Comment by gaeljw** [comment]
I observed the same behavior and thought it was deliberate for some reason but I'd like this to be fixed if this is actually a bug. It's confusing that labels without includeSelectors are not propagated to Deployments.

**Comment by KnVerey** [comment]
Fixed by #4209, docs in https://github.com/kubernetes-sigs/cli-experimental/pull/268 will merge any minute and be available at https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/labels/
